City-of-Helsinki/helsinki-paatokset
# PP-210: Issue page fixes

- Faulty diary number replaced with issue index number
- Margin / padding / fontsize +weight overhaul
- Attachments now have index numbers
- Show confidentiality reason if it exists

To test:
- Run `make drush-updb`
- Run paatokset_attachments -migration (`drush migrate-import paatokset_attachments`). If you haven't ran agenda_items and issues -migrations, run those too.
- View an issue page (eg. https://helsinki-paatokset.docker.so/fi/paatokset-issue/58243). Check that everything works and is in line with the sketch
